server/data/Users.txt

```diff
@@ -1,7 +1,6 @@
 2eda5ab0-6a63-11ea-a4b4-ebd4268cc60a,null,false,true,Jim,hunter123,Tim@tim.com,212,player_icon_1,4,10,2020-04-05T20:35:54.107Z,2020-04-06T18:28:32.603Z,2020-03-17T00:00:00.000Z,false,
 2eda5ab0-6a63-11ea-a4b4-ebd4268cc61b,null,false,false,Tim,hunter1234,Jim@Jim.com,5234,player_icon_1,5,12,2020-03-17T14:15:41.727Z,2020-04-06T18:28:32.603Z,2020-03-17T00:00:00.000Z,false,
-948bf810-71e2-11ea-a8e0-73473bc162b1,null,false,false,dev,dev,dev@gmail.com,13901,player_icon_1,0,0,2020-04-16T00:27:57.206Z,2020-04-16T00:19:28.496Z,2020-03-29T17:27:35.698Z,false,
+948bf810-71e2-11ea-a8e0-73473bc162b1,nMZ1Z_humS4vFcQNAAAA,true,false,dev,dev,dev@gmail.com,13901,player_icon_1,0,0,2020-04-16T15:53:17.897Z,2020-04-16T15:53:17.893Z,2020-03-29T17:27:35.698Z,false,
```

Comment:
could you reset this sign in boolean to false?

Reply:
yeah just caught that, its fixed